freqtrade/rpc/telegram.py

```diff
@@ -448,7 +448,8 @@ def _status(self, update: Update, context: CallbackContext) -> None:
                     "*Current Pair:* {pair}",
                     "*Amount:* `{amount} ({stake_amount} {base_currency})`",
                     "*Entry Tag:* `{buy_tag}`" if r['buy_tag'] else "",
-                    "*Exit Reason:* `{sell_reason}`" if r['sell_reason'] else "",
+                    "*Exit Reason:* `{sell_reason}`"
+                    if (r['sell_reason'] and not r['is_open']) else "",
```

Comment:
I don't think checking for `is_open` is correct (basically, it'll not have the impact you hope it does).

is_open is only set to False in one place - in `trade.close()` - which is only called once the trade is "really really closed" (when the order is filled). 

after that, there'll be no more checking for cancel and whatnot - so the trade is technically closed - and won't be reopened unless you mess with the database manually.

Reply:
So the story was
Someone is on gateio, limit sell order. A sell order was fully canceled, but if you check /status, it would show you the sell reason, because apparently canceled sell order don't remove the sell reason from the db.

So my intention is only show the sell reason once a trade is really really closed. So I believe the change proposed is doing that?

Reply:
It's not showing it while the sell-order is open (so you'll see "we're selling" - but not why).

Now that's obviously a point of discussion - if that's intended it's fine with me - but from the description - it didn't appear to be.

Reply:
Fair, I didn't think of that case. So what would be the best behaviour?

If a sell trade still open, show the reason. If a sell trade closed but the trade still not really really closed, hide the sell reason?

Reply:
I've moved the fix to the "order-cancellation" now so when the order is cancelled, sell_reason is reset.
this is better not just for this notification, but also for freqUI - as well as to keep the database clean.

Reply:
thanks